codegen/src/test/resources/software/amazon/awssdk/codegen/poet/model/sharedstream/streamdeathsinputeventstream.java

```diff
@@ -0,0 +1,79 @@
+package software.amazon.awssdk.services.sharedeventstream.model;
+
+import java.util.EnumSet;
+import java.util.Map;
+import java.util.Set;
+import software.amazon.awssdk.annotations.Generated;
+import software.amazon.awssdk.annotations.SdkPublicApi;
+import software.amazon.awssdk.services.sharedeventstream.model.streamdeathsinputeventstream.DefaultPerson;
+import software.amazon.awssdk.utils.internal.EnumUtils;
+
+/**
+ * Base interface for all event types in StreamDeathsInputEventStream.
+ */
+@Generated("software.amazon.awssdk:codegen")
+@SdkPublicApi
+public interface StreamDeathsInputEventStream {
+    /**
+     * Create a builder for the {@code Person} event type for this stream.
+     */
+    static PersonStreamDeathsInputEventStream.Builder personBuilder() {
+        return DefaultPerson.builder();
```

Comment:
Can we have tests to verify the code for `DefaultPerson`?

Reply:
The `SharedStreamAwsModelSpecTest` is testing only the interfaces, but `EventModelSpecTest` (which uses a different model) tests tests the `Default<Shape>` generation (but not the interfaces) for event stream event shapes - between the two I think we get decent coverage of all of the files we're generating.  

It might be a little tricky, but the SharedStreamAwsModelSpecTest could probably be updated to test the implementations (ie `Default<shape>.java`) as well but we would need to add files for all of the shapes I think.  What do you think?

Reply:
Yeah let's do that please. I don't think we need to do it for every renamed shape, just for a sample.